port/unix/omrsysinfo.c

```diff
@@ -299,6 +311,12 @@ struct {
 	{ "memory", OMR_CGROUP_SUBSYSTEM_MEMORY }
 };
 
+const char *OMRContainerNames[OMR_CONTAINER_NUM_CONTAINERS] = {
```

Comment:
As per comment above, this belongs in a separate changeset or should wait until we have more examples of container systems to build the api on.